pkg/tlsclientconfig/tlsclientconfig_test.go

```diff
@@ -90,7 +90,7 @@ func TestSetupCertificates(t *testing.T) {
 	// Unreadable CA certificate
 	tlsc = tls.Config{}
 	err = SetupCertificates("testdata/unreadable-ca", &tlsc)
-	assert.Error(t, err)
+	assert.NoError(t, err)
```

Comment:
Please add also tests for `tlsc` having the expected contents (either reuse the “RootCAs include SystemCertPool” condition from earlier in the test, or rearrange `SetupCertificates` to keep `tlsc.RootCAs == nil` and test for that.

Reply:
Hmm, I don't think I quite understand what you're describing in that first option, so made changes to the second.